components/org.wso2.carbon.identity.application.authenticator.fido2/src/main/java/org/wso2/carbon/identity/application/authenticator/fido2/core/WebAuthnService.java

```diff
@@ -101,6 +95,8 @@
 
     private static ArrayList origins = null;
 
+    private static final String timeout = IdentityUtil.getProperty("FIDO.UserResponseTimeout");
```

Comment:
The variable name `timeout` should be more readable.

Reply:
Addressed with https://github.com/wso2-extensions/identity-local-auth-fido/pull/58/commits/b5d30d70b50ddc46d7d07c2043fc0f0d876d94a2